ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-commands.sh

```diff
@@ -40,6 +42,7 @@ cat > packet-setup.yaml <<-EOF
         facility: any
         wait_for_public_IPv: 4
         state: active
+        tags: "{{ lookup('env', 'PROW_JOB_ID') }}"
```

Comment:
On a second review, probably it's better to add the following env vars:
- `PULL_NUMBER`
- `JOB_NAME`
With `BUILD_ID` already present in the hostname, a person will have enough (required) information to rebuild the original job link

Reply:
Done